pkg/backup/schedule_exec.go

```diff
@@ -578,6 +588,12 @@ func init() {
 						Measurement: "Jobs",
 						Unit:        metric.Unit_TIMESTAMP_SEC,
 					}),
+					RpoTenantMetric: metric.NewExportedGaugeVec(metric.Metadata{
+						Name:        "schedules.BACKUP.tenant-last-completed-time",
+						Help:        "The Unix timestamp of the most recently completed backup by host schedule for the tenant",
```

Comment:
Something like "The unix timestamp of the most recently completed tenant backup by a host schedule specified as maintaining this metric" would be more fitting and similar to the existing description.

Reply:
I like: "The unix timestamp of the most recently completed host scheduled backup by virtual cluster specified as maintaining this metric"